cookbooks/aws-parallelcluster-environment/files/default/ec2_udev_rules/ec2_dev_2_volid.py

```diff
@@ -70,6 +60,55 @@ def main():
     else:
         dev = dev.replace("xvd", "sd")
         dev = "/dev/" + dev
+    return dev
+
+
+def parse_proxy_config():
```

Comment:
I decided not to test this function because it seemed to just be calling lots of other functions without much logic in that function itself. Let me know if you think I should test this too though!